### PR TITLE
[plugin.audio.sverigesradio] 2.2.0

### PR DESCRIPTION
**Pull request by dodoadoodoo**
### Description
Added paging, stability fixes
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.audio.sverigesradio/resources/language/Swedish/strings.xml [hunk 1]
```diff
@@ -14,9 +14,12 @@
     <string id="32000">Inställningar</string>
```
Comment (enen92) [inline plugin.audio.sverigesradio/resources/language/Swedish/strings.xml]: strings.xml is not compatible with Matrix, support has been fully removed. Please adapt the new gettext format:

https://kodi.wiki/view/Language_support

File: plugin.audio.sverigesradio/addon.py [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
 from datetime import datetime
+from datetime import timedelta
+from datetime import date
+from distutils.util import strtobool
+import sys
 import itertools
 import requests
 from xbmcswift2 import Plugin
 
+import pprint
```
Comment (enen92) [inline plugin.audio.sverigesradio/addon.py]: unused import

**Comment by enen92** [comment]
@dodoadoodoo updates must be both python2 and python3 compatible. There's a pending PR for xbmcswift2 to add python3 compatibility: https://github.com/xbmc/repo-scripts/pull/1222

**Comment by dodoadoodoo** [comment]
> @dodoadoodoo updates must be both python2 and python3 compatible. There's a pending PR for xbmcswift2 to add python3 compatibility: [xbmc/repo-scripts#1222](https://github.com/xbmc/repo-scripts/pull/1222)

Right. Does this new version support strings.po as well?

File: plugin.audio.sverigesradio/resources/language/Swedish/strings.xml (hunk 1 above)
Comment (dodoadoodoo) [reply plugin.audio.sverigesradio/resources/language/Swedish/strings.xml]: Any chance of merging anyways, seeing as the current version is broken (poor error handling on my part)

File: plugin.audio.sverigesradio/resources/language/resource.language.en_gb/strings.po [hunk 3]
```diff
@@ -0,0 +1,115 @@
+# XBMC Media Center language file
```
Comment (enen92) [inline plugin.audio.sverigesradio/resources/language/resource.language.en_gb/strings.po]: please change to /resources/language/English/strings.po  for backward compatibility with gotham

File: plugin.audio.sverigesradio/resources/language/resource.language.sv_se/strings.po [hunk 4]
```diff
@@ -0,0 +1,106 @@
+# XBMC Media Center language file
```
Comment (enen92) [inline plugin.audio.sverigesradio/resources/language/resource.language.sv_se/strings.po]: resources/language/Swedish/strings.po

Comment (dodoadoodoo) [reply plugin.audio.sverigesradio/resources/language/resource.language.sv_se/strings.po]: Done

File: plugin.audio.sverigesradio/resources/language/resource.language.en_gb/strings.po (hunk 3 above)
Comment (dodoadoodoo) [reply plugin.audio.sverigesradio/resources/language/resource.language.en_gb/strings.po]: Done

**Comment by enen92** [comment]
@dodoadoodoo please squash your commits into a single one named "plugin.audio.sverigesradio] 2.2.0" for merging

**Comment by dodoadoodoo** [comment]
> @dodoadoodoo please squash your commits into a single one named "plugin.audio.sverigesradio] 2.2.0" for merging

I was afraid it would come to this, as git squash is not my friend. I think it will be quicker for me to submit a 2.2.1 request with identical content. Your view?

**Comment by Rechi** [comment]
To fix your pull request execute the following commands (origin is your repo, upstream is the xbmc repo)

```
git fetch upstream
git checkout gotham
git reset --hard upstream/gotham
git checkout origin/gotham -- plugin.audio.sverigesradio/
git commit -m '[plugin.audio.sverigesradio] 2.2.0'
git push -f origin gotham
```

**Comment by enen92** [comment]
@dodoadoodoo it's okay to submit a new PR, I'll handle it. Either way learning how to squash your commits is always the best option ^^

**Comment by dodoadoodoo** [comment]
Right, I think that should do it. Thanks for your patience and all help!

**Comment by enen92** [comment]
Thanks for adjusting

**Pull request merged**